introduction.md

```diff
@@ -0,0 +1 @@
+Starting to work with fork feature in GitHub
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
I have just commit the extra info

Reply:
@codeschool-kiddo I have followed instructions but the "pull request" is pending and the callenges is failing for this reason